Model/Order/Notifier/MailSender.php

```diff
@@ -14,17 +14,25 @@
 use Magento\Sales\Api\Data\OrderInterface;
 use Magento\Store\Model\StoreManagerInterface;
 use Opengento\Gdpr\Model\Notifier\AbstractMailSender;
+use Psr\Log\LoggerInterface;
 
 final class MailSender extends AbstractMailSender implements SenderInterface
 {
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
```

Comment:
The latest changes uses typed properties of php7.4. Please could you keep them? The composer.json php restriction is set to min php:7.4